webrender_traits/src/types.rs

```diff
@@ -372,6 +372,35 @@ impl Into<f64> for SubpixelOffset {
 }
 
 #[derive(Clone, Hash, PartialEq, Eq, Debug, Deserialize, Serialize, Ord, PartialOrd)]
+pub struct SubpixelPoint {
+    pub x: SubpixelOffset,
+    pub y: SubpixelOffset,
+}
+
+impl SubpixelPoint {
+    pub fn new(point: Point2D<f32>,
+               render_mode: FontRenderMode) -> SubpixelPoint {
+        SubpixelPoint {
+            x: render_mode.subpixel_quantize_offset(point.x),
+            y: render_mode.subpixel_quantize_offset(point.y),
+        }
+    }
+
+    pub fn x_offset(&self) -> f64 {
```

Comment:
Having `fn to_f64(&self) -> (f64, f64)` would be simpler both here and at the caller sites